package.json

```diff
@@ -4,16 +4,18 @@
   "description": "bugsnag logger for book",
   "main": "index.js",
   "dependencies": {
-    "bugsnag": "1.6.3",
+    "bugsnag": "1.7.0",
     "nock": "0.36.2",
     "xtend": "3.0.0"
   },
   "devDependencies": {
+    "book": "1.3.1",
     "mocha": "1.20.1",
-    "book": "1.3.1"
+    "nsp": "^2.6.1"
```

Comment:
What is nsp and why does it need to be a dependency? Seems like some tool that can be used without adding to every single module out there. Would be +1 for bugsnag update and -1 for adding nsp.